tests/integration_tests/charts/api_tests.py

```diff
@@ -933,15 +933,27 @@ def test_admin_gets_filtered_energy_slices(self):
                 }
             ],
             "keys": ["none"],
-            "columns": ["slice_name"],
+            "columns": ["slice_name", "description", "table.table_name"],
         }
         self.login(username="admin")
 
         uri = f"api/v1/chart/?q={prison.dumps(arguments)}"
         rv = self.get_assert_metric(uri, "get_list")
-        self.assertEqual(rv.status_code, 200)
-        data = json.loads(rv.data.decode("utf-8"))
-        self.assertEqual(data["count"], 8)
+        data = rv.json
+        assert rv.status_code == 200
+        assert data["count"] > 0
+        for chart in data["result"]:
+            print(chart)
+            assert (
+                "energy"
+                in " ".join(
+                    [
+                        chart["slice_name"] or "",
+                        chart["description"] or "",
+                        chart["table"]["table_name"] or "",
+                    ]
+                ).lower()
+            )
```

Comment:
Bycatch: asserting for entity counts is probably not a good idea---as the test database can have different things for various reasons. We should test against the actual underlying function itself----which, in this case, is filtering by a couple of text fields.